buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
             "PB_DockerTag": "centos-7-d485f41-20173404063424",
             "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols",
             "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
-            "PB_SyncArguments": "-p -- /p:ArchGroup=x64",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=x64 /p:OverridePackageSource=$(PB_AzureFeedUrl)",
```

Comment:
It seems like it would be worthwhile to make `BuildParameters` entries for this and add support in the build definitions rather than changing every `PB_SyncArguments` (and some `PB_BuildArguments`).

Reply:
Wasn't sure if that was supported, will wait for a passing build

Reply:
I wouldn't wait on making / validating this suggestion.  I think it is a good suggestion and should be done, but I'd merge and then clean this up tomorrow.